### PR TITLE
Update README.md

### LINKED ISSUE #5119: Typos in README.md
There are some typos in the documentation ( some grammatical errors and some typing mistakes)

### PR DESCRIPTION
**Pull request by sg7801**
Fixes https://github.com/backdrop/backdrop-issues/issues/5119
I have corrected few errors in the documentation.

### CODE REVIEW

**Comment by indigoxela** [comment]
@sg7801 thank you for fixing those typos! :pray:  I do have a small request, can you please add the link to the issue this PR fixes? I've no idea which one this relates to.

Just add `Fixes https://the full link to the issue in the queue here` where you now posted "I have corrected few errors in the documentation."

The word "Fixes" will then automatically link back to the PR from the related issue.

**Comment by sg7801** [comment]
I have updated the issue associated with this pull request.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Requirements
   strongly recommend that you use a
   [supported version of PHP](https://secure.php.net/supported-versions.php).
 - MySQL 5.0.15 or higher with PDO enabled
-- Apache (recommended) or Nginx web server
+- Apache (recommended) or Nginx webserver
```
Comment (klonos) [inline README.md]: @sg7801 I agree with the rest of the changes, but not with this one. Please see: 

- https://en.wikipedia.org/wiki/Nginx
  > Nginx, stylized as NGINX, nginx or NginX, is a **_web server_** that...
- https://www.nginx.com
  > NGINX | High Performance Load Balancer, **_Web Server_**
- https://en.wikipedia.org/wiki/Apache_HTTP_Server
  > The Apache HTTP Server, colloquially called Apache, is a free and open-source cross-platform **_web server_** software...

**Review by klonos — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sg7801) [reply README.md:16]: Sure! I will change "webserver" back to how it was!

Comment (sg7801) [reply README.md]: @klonos I have changed "webserver" as suggested by you

File: README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ join Backdrop's developer community by
 
 When submitting a pull request, please make sure there is an open issue in the
 [issue tracker](https://github.com/backdrop/backdrop-issues/issues) and
-reference it in the description on the pull request.
```
Comment (ghost) [inline README.md]: I don't see a problem with this original wording.

Comment (sg7801) [reply README.md]: @BWPanda I thought "description of pull request" would sound better than "description on pull request" in English

Comment (sg7801) [reply README.md:58]: If you say I would revert this change to orginial wording

**Comment by sg7801** [comment]
I have made the changes suggested

**Comment by sg7801** [comment]
I have made the suggested changes

File: README.md (hunk 2 above)
Comment (herbdool) [reply README.md]: I think the original is more accurate - there is a "description" field on github's PR screen.

**Comment by ghost** [comment]
Thanks @sg7801! Merged into 1.x and 1.20.x.

**Pull request merged**